docs/commands/emoji.md

```diff
@@ -0,0 +1,35 @@
+!!! info
+    
+    These commands can be executed by Admins and Bot Manager only.
+
+# Emoji Commands
```

Comment:
I think heading should be on top and lots of commands need a bit more explanation because this same things is available in `help` command anyways.